python/prophet/models.py

```diff
@@ -72,11 +69,8 @@ def get_type():
 
     def load_model(self):
         import cmdstanpy
-        model_file = pkg_resources.resource_filename(
-            'prophet',
-            'stan_model/prophet_model.bin',
-        )
-        return cmdstanpy.CmdStanModel(exe_file=model_file)
+        model_file = importlib_resources.files("prophet").joinpath("stan_model/prophet_model.bin")
```

Comment:
The cannonical way to join multiple paths is to either pass them as multiple arguments like `.joinpath("stan_model", "prophet_model.bin")`, or to use the `/` operator, like 
```suggestion
        model_file = importlib_resources.files("prophet") / "stan_model" / "prophet_model.bin"
```